src/components/Pagination/Pagination.js

```diff
@@ -180,8 +221,7 @@ export default class Pagination extends Component {
           ) : (
             <TextInput
               id={`bx-pagination-input-${inputId}`}
-              placeholder="0"
```

Comment:
Did you want to kill the `placeholder` here?

Reply:
OK - Seems that this prop is for `<input>` placeholder, which won't interfere user input?

Reply:
No - placeholder is for when `<input>` component is empty and doesn't interfere with user input.

Reply:
Placeholder was not getting used before since page # had to be > 0... We don't want placeholder since if user deletes page # it will put 0 in `input` instead of keeping it empty

Reply:
👍 Thank you for articulating!